docs/blog/posts/2022-11-14_installing_software.md

```diff
@@ -0,0 +1,104 @@
+---
+date:
+  created: 2022-11-14
+categories:
+    - Yen
+    - Software
+authors:
+    - nrapstin
+---
+# Installing Software on the Yens
+
+As a yen user, you can install your own custom software in your home directory or any location where you have permissions
```

Comment:
I think we agreed on convention to use `yen` every time (not Yen, yen, or other variations)

Reply:
done